src/views/RoomView/RoomView.jsx

```diff
@@ -118,8 +132,31 @@ class RoomView extends Component {
     this.props.createCard(room.url, cardInput, 1);
   };
 
+  isAdmin = () => {
```

Comment:
이런 경우는 JavaScript `some` method를 사용하면 편합니다.

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/some


```suggestion
isAdmin = () => {
    const { me, room } = this.props;
    if (room) {
      return room.admins.some(admin => admin._id === me._id)
    }
    return false;
  }
```

Reply:
수정완료